modules/mediawiki/manifests/deploy.pp

```diff
@@ -70,6 +70,51 @@
         mode   => '0755',
     }
 
+    file { '/srv/mediawiki-staging/patches':
+        ensure  => 'directory',
+        owner   => 'www-data',
+        group   => 'www-data',
+        mode    => '0755',
+        require => File['/srv/mediawiki-staging'],
+    }
+
+    file { '/srv/mediawiki-staging/patches/public.json':
+        ensure  => 'file',
+        owner   => 'www-data',
+        group   => 'www-data',
+        mode    => '0644',
+        source  => 'puppet:///modules/mediawiki/patches/public.json',
```

Comment:
I wonder if it would be better to do these in mediawiki-repos so mediawiki-admins can push without ops review?

Reply:
That's a pretty good idea, I'll take a look at that soon.

Reply:
* @The-Voidwalker #3940 / miraheze/mediawiki-repos#35

Reply:
> * @The-Voidwalker [Use mediawiki-repos for public patches #3940](https://github.com/miraheze/puppet/pull/3940) / [Add patches directory mediawiki-repos#35](https://github.com/miraheze/mediawiki-repos/pull/35)

@Universal-Omega Is there anything else blocking this PR that needs to be added for merging and deployment besides #3940 ?